### ISSUE TITLE
Using DeLeo instead of House average

### ISSUE THREAD

**Issue by JonathanCohn** [aholachek/legislator-scorecard#33]
Got a recommendation this morning to use DeLeo's score instead of the House average for reps in the score card because it's more illustrative and doesn't make it seem like all of the alway-vote-with-DeLeo folks are "above average." 

Thoughts, @harmonywho?

**Comment by harmonywho** [comment]
Welll, personally, I like having the SPeaker as the standard. I think that it speaks to the precise problem in MA leg, which is not partisan, it's Leadership and The Way We Do Business Around Here. 

So.... I would vote for that change. 

In the spirit of parallelism, the Republican side could be "Minority Leader" score?

**Comment by JonathanCohn** [comment]
Yeah, I think we could do that for both sides in the House. The Senate is different because Stan barely votes.

**Comment by harmonywho** [comment]
[Bradley Jones](https://malegislature.gov/Legislators/Profile/BHJ1/Biography) 
is the minority leader. 

[Bob DeLeo](https://malegislature.gov/Legislators/Profile/RAD1) is the Speaker
